blocks/commerce-checkout/commerce-checkout.js

```diff
@@ -586,6 +587,7 @@ export default async function decorate(block) {
 
   async function initializeCheckout(data) {
     removeEmptyCart();
+    await initReCaptcha(0);
```

Comment:
If reCaptcha is not enabled for `Place Order`, this will return immediately, so there is no impact on performance here.
See https://github.com/adobe-commerce/StorefrontSDK/blob/01bb3feb336a876e4a616e37675ee79130e71a10/packages/recaptcha/src/index.ts#L167 for reference.